packages/rocketchat-2fa/client/accountSecurity.html

```diff
@@ -45,6 +45,7 @@ <h1>{{_ "Backup_codes"}}</h1>
 					{{/if}}
 				{{/if}}
 			</form>
+			{{> personalAccessTokens }}
```

Comment:
instead of adding this at `Security` page, I think it should have its own sub-menu.
this would also allow not writing it as a package, but as a [meteor module](https://docs.meteor.com/packages/modules.html)